models/ClimateSensor.json

```diff
@@ -0,0 +1,89 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:com:example:ClimateSensor;1",
+  "@type": "Interface",
+  "displayName": "Climate Sensor",
+  "description": "Provides functionality to report temperature, humidity, pressure",
+  "comment": "Requires temperature, humidity and pressure sensors.",
+  "contents": [
+    {
+      "@type": "Property",
+      "displayName": "Update interval",
+      "description": "Interval of telemetry updates in seconds",
+      "name": "updateInterval",
+      "writable": true,
+      "schema": "integer"
+    },
+    {
+      "@type": [
+        "Telemetry",
+        "Temperature"
+      ],
+      "description": "Current temperature on the device",
+      "displayName": "Temperature",
+      "name": "temperature",
+      "schema": "double",
+      "unit": "degreeCelsius"
+    },
+    {
+      "@type": [
+        "Telemetry",
+        "Humidity"
```

Comment:
>Humidity [](start = 9, length = 8)

You might mean for this semantic type to be RelativeHumidity. I'm not sure--check the different semantic types in the DTDL v2 language spec.

Reply:
RelativeHumidity is `unitless`. I was looking for good examples with educational purposes. Unitless may add confusion. For now I'm leaving the Humidity semantic type.